### ISSUE TITLE
Sprint 3 – Sprint Planning(12-12-2016)

### ISSUE THREAD

**Issue by juzplayplay** [mewind/RSDSupermarketPOS#14]
Hi guys, today will be the first day of the Sprint 3. I'm the scrum master for this sprint. For this sprin we will mainly focus on Payment module, did everyone of you understand all the user stories?

**Comment by mewind** [comment]
Yes, I did read through all the user stories and roughly know what we should do in this sprint. Since we have already linked the gridview in Payment tab with the products in cart during last Sprint, I think it should be no problem for us to do the next part.

**Comment by lkc442012** [comment]
Hmmm, in my opinion the function of Payment will be dealing with database a lot, therefore we have to be more careful when developing this module

**Comment by huylin25** [comment]
Yes I agree with KarCheng, because in the user story require us to actually update the quantity of products selected in the cart, and remove the item from the cart, therefore we need to keep on updating the database with the latest data

**Comment by mewind** [comment]
Ya, maybe this will be the challenging part in this sprint and need us to put more effort on it. Moreover, I think we will need to design another form which showing the payment details after each successful payment and ask the user whether to print the receipt, in order to provide them some confirmation after each payment

**Comment by juzplayplay** [comment]
I think maybe you guys should actually automatically log out the current user account everytime when they successfully done their payment to prevent user forgot to logout manually and the next user can misuse the system with their account.

**Comment by huylin25** [comment]
Okay, this will be the must for the security purpose.

**Comment by lkc442012** [comment]
This should be no problem since we just need to write a few lines of code but can protect the user account, thanks for reminding it!

**Comment by mewind** [comment]
Furthermore I think we should allow the user to cancel the payment, once they cancel the payment, the particular Sales ID in database will be deleted and the items in the cart will also be removed, this will be more convenient if the user want to remove everything from the cart instead of doing it one by one manually

**Comment by juzplayplay** [comment]
I think this is what most of the user expect to have, this will provide a more helpful function to them

**Comment by lkc442012** [comment]
So, in the calculation part we just need to have total of the price of products selected ?

**Comment by huylin25** [comment]
I think should also include GST induced to the subtotal of all the products selected.

**Comment by mewind** [comment]
Oh ya, we also need to check the identity of the user, if the user is either a member or staff of the supermarket they can enjoy 10% of discount, so we should also display the discount amount.

**Comment by lkc442012** [comment]
Okay, so that means we will need to have the Grandtotal which the formula should be (SUBTOTAL + GST)-DISCOUNTED price?

**Comment by huylin25** [comment]
Yes,  I think this will provide a clearer breakdown for the user to check on the payment amount.

**Comment by mewind** [comment]
At the same time user will have less doubts on the payment amount.

**Comment by juzplayplay** [comment]
Okay, so do you guys have any other ideas or problem to be discussed? If no, I think we can start to allocate the tasks.

**Comment by lkc442012** [comment]
I think should be no more.

**Comment by juzplayplay** [comment]
Okay, KarCheng, can you develop the calculation part? Which includes subtotal, GST, discounted amount and the grand total?

**Comment by lkc442012** [comment]
I think should be okay, I'll discuss with you guys if I have any problem.

**Comment by lkc442012** [comment]
I think I'll also handle the cancel Payment part since it should also be a quite simple task

**Comment by juzplayplay** [comment]
Sure ,no problem.

**Comment by mewind** [comment]
So, the Update Payment, show Payment Details, and logout after each successful payment will handle by HuyLin and me?

**Comment by juzplayplay** [comment]
Yes you're right, but we all will try our best to help if you guys have any problem.

**Comment by huylin25** [comment]
Okay, we will work together to make things done.

**Comment by juzplayplay** [comment]
So we will dismiss the first meeting of Sprint 3 now ? Any objection?

**Comment by huylin25** [comment]
Nope, thanks Raymond for conducting the meeting!

**Comment by juzplayplay** [comment]
You're welcome
